fastlane/Constants/Constant.swift

```diff
@@ -76,6 +78,11 @@ extension Constant {
             case .production: return "\(googleServiceFolder)/Production/\(infoName)"
             }
         }
+
+        var dsymPath: String {
+            let outputDirectoryURL = URL(fileURLWithPath: Constant.outputPath)
+            return outputDirectoryURL.appendingPathComponent(productName + ".app.dSYM.zip").relativePath
```

Comment:
I think we can turn this part into a type constant `".dSYM.zip"`  🙏

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/361/commits/c654fb72bf5aa6f942520f73fb7b391b953a3e58